pages/ballot/index.tsx

```diff
@@ -25,6 +25,59 @@ const Scroller = dynamic(() => import("components/Scroller/Scroller"), {
   ssr: false,
 });
 
+const Race = ({ race, itemId, incumbentId } : { race: RaceResult, itemId: string, incumbentId: string | undefined }) => {
```

Comment:
nice extraction.  i think we should start pulling these into separate files pretty soon